File: ProyectoPrograIV/ProyectoPrograIV/Scripts/app/Usuarios.js
$(document).on("ready", function () {
    ObtenerTodos();
})

//Obtener todas los estudiantes
function ObtenerTodos() {
    var item = "";
    $('#tblLista tbody').html('');
    $.getJSON('/api/Usuarios', function (data) {
        $.each(data, function (key, value) {
            item += "<tr><td>" + value.UsuID + "</td><td>" + value.UsuNombre + "</td><td>" + value.UsuGenero + "</td><td>" + value.UsuCorreo + "</td><td>" + value.UsuTelefono + "</td><td>" + value.UsuRol + "</td></tr>";
        });
        $('#tblLista tbody').append(item);
    });
};

File: ProyectoPrograIV/ProyectoPrograIV/Models/Usuarios.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace ProyectoPrograIV.Models
{
    public class Usuarios
    {
        public int UsuID { get; set; }
        public string UsuNombre { get; set; }
        public char UsuGenero { get; set; }
        public string UsuCorreo { get; set; }
        public string UsuTelefono { get; set; }
        public string UsuRol { get; set; }
    }
}